config/locales/es_core.yml

```diff
@@ -5,4 +5,10 @@ es:
         explanation: Por favor, indica el motivo por el que deseas eliminar tu cuenta (opcional).
     notifications_settings:
       show:
-        newsletter_notifications: Quiero recibir informació sobre la actividad relevante y las novedades del portal.
\ No newline at end of file
+        newsletter_notifications: Quiero recibir informació sobre la actividad relevante y las novedades del portal.
```

Comment:
ca_core i ex_core tenen claus diferents? aquí hi hauria d'haver les claus de share no?

Reply:
Hi ha caós força gros amb els locales, hem de mirar de centralitzar-ho tot.